pkg/dbal/DbalContext.php

```diff
@@ -170,7 +170,7 @@ public function createDataBaseTable()
             return;
         }
 
-        if ($this->getDbalConnection()->getDatabasePlatform()->hasNativeGuidType()) {
+        if (!$this->getDbalConnection()->getDatabasePlatform()->hasNativeGuidType()) {
```

Comment:
I guess this change would break MySql. Could you remove it?

Reply:
I have removed the check. That check should not be necessary in the first place because Doctrine will use the UUID column type if available, otherwise fall back to VARCHAR. One thing to take note of is that it may be necessary to add some code to #280 to detect and migrate the column type if you decide to merge that PR.